projects/plugins/jetpack/extensions/blocks/blogroll/edit.js

```diff
@@ -37,8 +39,20 @@ export function BlogRollEdit( { className, attributes, setAttributes, clientId }
 		}
 	}, [ recommendations, load_placeholders, setAttributes, clientId, replaceInnerBlocks ] );
 
+	useEffect( () => {
+		setAdditionalClasses( [
+			...( ! show_avatar ? [ 'hide-avatar' ] : [] ),
+			...( ! show_description ? [ 'hide-description' ] : [] ),
+			...( ! show_subscribe_button ? [ 'hide-subscribe-button' ] : [] ),
+		] );
+	}, [ show_avatar, show_description, show_subscribe_button ] );
+
+	const blockProps = useBlockProps( {
+		className: classNames( className, ...additionalClasses ),
+	} );
```

Comment:
No need to store additional classes in a state. They're derivative.

```suggestion
	const blockProps = useBlockProps( {
		className: classNames( className, {
			'hide-avatar': ! show_avatar,
			'hide-description': ! show_description,
			'hide-subscribe-button': ! show_subscribe_button
		} ),
	} );
```

Reply:
agh yes fixing this